### PR TITLE
fix: fix broken image signing 

### PR DESCRIPTION
**Pull request by Kavindu-Dodan**
## This PR

Fixes image signing issue and sign with digest

### CODE REVIEW

File: .github/workflows/release-please.yaml [hunk 1]
```diff
@@ -77,13 +78,20 @@ jobs:
...
+        outputs:
+          image_digest: ${{ steps.build.outputs.digest }}
 
+  container-signing:
+    needs: container-release
+    runs-on: ubuntu-latest
+    if: ${{ needs.release-please.outputs.release_created }}
+    steps:
       - name: Install Cosign
         uses: sigstore/cosign-installer@main
```
Comment (beeme1mr) [inline .github/workflows/release-please.yaml:90]: Can you explicitly set the version or commit sha here please?

Comment (toddbaert) [reply .github/workflows/release-please.yaml:90]: ^

Comment (toddbaert) [reply .github/workflows/release-please.yaml:90]: I vote 2

File: .github/workflows/release-please.yaml [hunk 2]
```diff
@@ -77,13 +78,20 @@ jobs:
...
             COMMIT=${{ github.sha }}
             DATE=${{ steps.date.outputs.date }}
+        outputs:
+          image_digest: ${{ steps.build.outputs.digest }}
 
+  container-signing:
+    needs: container-release
+    runs-on: ubuntu-latest
+    if: ${{ needs.release-please.outputs.release_created }}
+    steps:
```
Comment (toddbaert) [inline .github/workflows/release-please.yaml:88]: Makes sense to put this in a new job.

File: .github/workflows/release-please.yaml (hunk 1 above)
Comment (Kavindu-Dodan) [reply .github/workflows/release-please.yaml]: I updated gh action to use digest instead of @main

**Review by toddbaert — APPROVED** [review]

**Pull request merged**